src/herbpy/herbrobot.py

```diff
@@ -298,7 +305,6 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
             self._say_action_client = SimpleActionClient('say', talker.msg.SayAction)
 
     def CloneBindings(self, parent):
-        from prpy import Cloned
```

Comment:
Nit: Should we really move this to the top. We can create a robot without needing clone functionality.